run_eval_model.py

```diff
@@ -124,6 +124,10 @@ def main():
         batch_tgt=dset.all_tgt(),
         batch_pred=all_pred,
     ))
+    print(DSET_OPTS[args.dset_name].bleu_score(
```

Comment:
This method only work on translation dataset but not arithmetic dataset.
Overload `batch_eval` can do the trick.